packages/itwin/map-layers/src/ui/widget/AttachLayerPopupButton.tsx

```diff
@@ -389,6 +388,7 @@ function AttachLayerPanel({ isOverlay, onLayerAttached, onHandleOutsideClick }:
           await MapLayerPreferences.deleteByName(source, iTwinId, iModelId);
           const msg = MapLayersUI.localization.getLocalizedString("mapLayers:CustomAttach.RemoveLayerDefSuccess", { layerName });
           IModelApp.notifications.outputMessage(new NotifyMessageDetails(OutputMessagePriority.Info, msg));
+        // eslint-disable-next-line unused-imports/no-unused-vars
```

Comment:
you could just delete err
```ts
catch {
    blah
}
```

Reply:
oop, okay, refactored all instances of this